linters/bandit.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2018 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+const cache = require('../cache')
+
+const report = require('../bandit/bandit_report')
+
+module.exports = class Bandit {
```

Comment:
Does javascript have abstract classes or interfaces the class would define?

Reply:
I'm not very familiar with "design patterns" in javascript, I did some research on this and it appears there isn't (we would need to switch to typescript for this).
I might try another approach based on delegation (the prototypal equivalent of inheritance) but I don't know if it would solve the problem of enforcing a common interface for all linter objects.

Reply:
See #102 for discussion